### PR TITLE
[18] The Software Developer conditionals

### PR DESCRIPTION
**Pull request by emmanuelidiaghe**
## Task: [18] The Software Developer conditionals
- did the second javascript task

### CODE REVIEW

**Comment by ToluAkin** [comment]
@emmanuelidiaghe In your previous pull request you were meant to remove images in your javascript folder. You should do that in this pull request.

**Comment by emmanuelidiaghe** [comment]
Oh that? I thought you referred to the screenshot I added to the pull request.
Alright, it’s done!

File: submissions/EmmanuelIdiaghe/javascript/index.js [hunk 1]
```diff
@@ -24,4 +24,24 @@ var productList = [
...
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+/*
+    Task: Print Numbers from 1 - 116 with conditionals
+*/
+
+const positiveDiff = (Math.abs(7 - 123));
+
+for (var counter = 1; counter <= positiveDiff; counter++) {
+    if(((counter % 3) == 0) && ((counter % 5) == 0)) {
```
Comment (adaugochi) [inline submissions/EmmanuelIdiaghe/javascript/index.js:37]: The brackets are too much. Doing this `(counter % 3 == 0 && counter % 5 == 0 )` will still work file I guess. Same for others too

**Comment by emmanuelidiaghe** [comment]
Fixed parentheses!

**Review by adaugochi — APPROVED** [review]

**Pull request merged**